dpnp/dpnp_iface_arraycreation.py

```diff
@@ -1370,29 +1378,35 @@ def zeros(shape, dtype=None, order='C'):
     [0.0, 0.0]
 
     """
+    if like is None:
+        _dtype = dtype if dtype is not None else dpnp.float64
+        return dpnp_container.zeros(shape,
+                                    dtype=_dtype,
+                                    order=order,
+                                    device=device,
+                                    usm_type=usm_type,
+                                    sycl_queue=sycl_queue)
 
-    if (not use_origin_backend()):
-        if order not in ('C', 'c', None):
-            pass
-        else:
-            _dtype = dtype if dtype is not None else dpnp.float64
-            result = dpnp_zeros(shape, _dtype).get_pyobj()
-
-            return result
-
-    return call_origin(numpy.zeros, shape, dtype=dtype, order=order)
+    return call_origin(numpy.zeros, shape, dtype=dtype, order=order, like=like)
 
 
 # numpy.zeros_like(a, dtype=None, order='K', subok=True, shape=None)
-def zeros_like(x1, dtype=None, order='C', subok=False, shape=None):
+def zeros_like(x1,
+               *,
+               dtype=None,
+               order="C",
+               subok=False,
+               shape=None,
+               device=None,
+               usm_type="device",
+               sycl_queue=None):
     """
     Return an array of zeros with the same shape and type as a given array.
 
     For full documentation refer to :obj:`numpy.zeros_like`.
 
     Limitations
     -----------
-    Parameter ``order`` is supported only with default value ``"C"``.
```

Comment:
Order is still limited with "C" and "F" in dpctl, and default is "C" while "K" in NumPy. I think we need to update the limitation, rather than to remove it.